### PR TITLE
Update small-medium-data-warehouse-content.md

### PR DESCRIPTION
**Pull request by galinagpolyakova**
@Ed Price to add image

### CODE REVIEW

**Review by EdPrice-MSFT — APPROVED** [review]
These tweaks look great

**Comment by EdPrice-MSFT** [comment]
#Sign-Off

**Comment by PRMerger15** [comment]
@EdPrice-MSFT: I'm sorry - only the author of this article, @, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by Jak-MS** [comment]
@galinagpolyakova @EdPrice-MSFT - at the moment the mix of bullets and numbers needs some clarification because it makes the information flow confusing. I'm going to make some change suggestions to address this issue. thanks.

File: docs/example-scenario/data/small-medium-data-warehouse-content.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This solution isn't recommended for:
 
 ![Diagram showing how legacy data can migrate and modernize with Azure Synapse, SQL Database, Data Lake Storage, and other services.](media/small-medium-data-warehouse/small-medium-data-warehouse.svg)
 
-1. Legacy SMB data warehouses might contain several types of data:
+- Legacy SMB data warehouses might contain several types of data:
```
Comment (Jak-MS) [inline docs/example-scenario/data/small-medium-data-warehouse-content.md:27]:
```suggestion
Legacy SMB data warehouses might contain several types of data:
```

File: docs/example-scenario/data/small-medium-data-warehouse-content.md [hunk 2]
```diff
@@ -36,18 +36,20 @@ This solution isn't recommended for:
...
    
    Serverless pools are ideal for:
    - Ad hoc data science explorations in T-SQL format.
    - Early prototyping for data warehouse entities.
    - Defining views that consumers can use, for example in Power BI, for scenarios that can tolerate performance lag.
-   
-   This method is an easy way to mesh Dynamics data sources, or build centralized BI dashboards on augmented datasets. You can bring the fused, processed data back into Dynamics and Power BI for further analysis.
 
-1. Azure Synapse is tightly integrated with potential consumers of your fused datasets, like Azure Machine Learning. Other consumers can include Power Apps,  Azure Logic Apps, Azure Functions apps, and Azure App Service web apps.
+- Azure Synapse is tightly integrated with potential consumers of your fused datasets, like Azure Machine Learning. Other consumers can include Power Apps,  Azure Logic Apps, Azure Functions apps, and Azure App Service web apps.
```
Comment (Jak-MS) [inline docs/example-scenario/data/small-medium-data-warehouse-content.md:52]:
```suggestion
Azure Synapse is tightly integrated with potential consumers of your fused datasets, like Azure Machine Learning. Other consumers can include Power Apps,  Azure Logic Apps, Azure Functions apps, and Azure App Service web apps.
```

**Review by Jak-MS — APPROVED** [review]

**Pull request merged**